contracts/LosslessControllerV3.sol

```diff
@@ -447,15 +482,18 @@ contract LosslessControllerV3 is Initializable, ContextUpgradeable, PausableUpgr
     /// @param recipient Address to lift the locks
     function _removeExpiredLocks (address recipient) private {
         LocksQueue storage queue;
-        queue = tokenScopedLockedFunds[msg.sender].queue[recipient];
+        queue = tokenScopedLockedFunds[ILERC20(msg.sender)].queue[recipient];
 
         uint i = queue.first;
+        uint256 lastItem = queue.last;
+
         ReceiveCheckpoint memory checkpoint = queue.lockedFunds[i];
 
-        while (checkpoint.timestamp <= block.timestamp && i <= queue.last) {
+        while (checkpoint.timestamp <= block.timestamp && i <= lastItem) {
             delete queue.lockedFunds[i];
             i += 1;
             checkpoint = queue.lockedFunds[i];
```

Comment:
I don't think CVF-458 is fixed. When i == lastItem, loop will be entered and i will be incremented by 1, then checkpoint = queue.lockedFunds[i] will try to access lastItem + 1 ?